docs/getting-started/gs-normalization-encoding.md

```diff
@@ -43,13 +43,15 @@ To normalize an email address, complete the following steps:
 
 ## Email Address Hash Encoding
 
-An email hash is a Base64-encoded SHA-256 hash of a normalized email address. The email address is first normalized, then hashed using the SHA-256 hashing algorithm, and then encoded using Base64 encoding.
+An email hash is a Base64-encoded SHA-256 hash of a normalized email address. The email address is first normalized, then hashed using the SHA-256 hashing algorithm, and the resulting hex value is encoded using Base64 encoding.
```

Comment:
Should be something like:

> the resulting bytes of the hash value (rather than their hex-encoded string representation) is encoded using Base64 encoding